mastering-plone/thirdparty_behaviors.rst

```diff
@@ -25,12 +25,9 @@ To use it add the name to your list of eggs in :file:`buildout.cfg`:
         ...
         collective.behavior.banner
 
-Even though :py:mod:`collective.behavior.banner` has been released on PyPI we will now act as if this addon exists on GitHub or has changes on GitHub that have not yet been released, but that you really want.
-
+Even though :py:mod:`collective.behavior.banner` has been released on PyPI we will now act as if this add-on exists on GitHub or has changes on GitHub that have not yet been released, but that you really want.
 This is not to annoy you.
-
 It happens surprisingly often if you work with new versions of Plone.
-
```

Comment:
This blank line should be restored to separate these blocks into two paragraphs.

Reply:
Done.